controllers/component_controller.go

```diff
@@ -308,6 +308,7 @@ func (r *ComponentReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 				return ctrl.Result{}, err
 			}
 
+			// If the DockerfileURL is set in the Git source - update any local references to the Dockerfile in the devfile to use the fully qualified URL
```

Comment:
probably want to place this somewhere close to the actual line

Reply:
Updated